server/config/index.js

```diff
@@ -41,10 +41,12 @@ nconf.defaults({
  * Convenience values
  */
 const isProduction = nconf.get('NODE_ENV') === 'production';
+const isRemoteDebug = nconf.get('REMOTE_DEBUG') === 'true';
 const useDevSources = nconf.get('DEV_SOURCES') === 'true';
 const publicDir = isProduction || useDevSources ? './dist' : '../client';
 
 nconf.set('IS_PRODUCTION', isProduction);
+nconf.set('IS_REMOTE_DEBUG', isRemoteDebug);
```

Comment:
I'm not keen on the use of `nconf.set` because it mutates global state. Could we either use `nconf.get('REMOTE_DEBUG') === true` or use a module to abstract this and expose it as a function?